tracer/src/Datadog.Trace/Configuration/IntegrationIds.cs

```diff
@@ -36,6 +36,8 @@ internal enum IntegrationIds
         AwsSqs,
         ILogger,
         Aerospike,
-        AzureFunctions
+        AzureFunctions,
+        Couchbase,
```

Comment:
This should probably be `Couchbase2`. Note this name is exposed to users and will be hard to change once released. It's how users configure per-integration settings (e.g. disable a specific integration).

See the comment above for `ElasticsearchNet`. We wrote that one first then added `ElasticsearchNet5`. Now we're stuck with `ElasticsearchNet` since we couldn't change it to `ElasticsearchNet6`.

Reply:
(we could also allow multiple names for the same integration, so that `ElasticsearchNet == ElasticsearchNet6`, but that's a separate topic 😅 )

Reply:
Ok changed it according to our discussion to have only one IntegrationId